data/deploy_types.yml

```diff
@@ -1,3 +1,7 @@
+qemu:
+  name: QEmu/KVM
+  download_platform: openstack
+  ext: qc2
```

Comment:
Can you move this down so that this file is sorted by name (i.e. between rhev and vsphere)?